SimDataFormats/CaloAnalysis/src/classes_def.xml

```diff
@@ -8,14 +9,37 @@
   <class name="CaloParticleRefVector"/>
   <class name="CaloParticleRefProd"/>
   <class name="CaloParticleContainer"/>
-  <class name="SimCluster"  ClassVersion="4">
-    <version ClassVersion="4" checksum="3868743425"/>
-    <version ClassVersion="3" checksum="1773780587"/>
+<class name="std::pair<edm::ProductID,int>" />
```

Comment:
This dictionary should be declared in `DataFormats/Provenance`. The package already declares dictionary for `std::pair<edm::ProductID, unsigned int>`, which makes me wonder if it would be feasible to use `unsigned int` here instead of `int`? (I did not look in where and howthis `pair` is used in this PR)

Reply:
In the end I've removed the part of the code that needed this dictionary, so I've removed it as well